docs/release/release-team.md

```diff
@@ -81,9 +81,11 @@ When assembling a release team, the release team lead should look for volunteers
 
 - Can commit to the amount of time required across the release cycle
 - Are enthusiastic about being on the release team
-- Are members of the Cluster Lifecycle SIG
+- Preferably are [members of the kubernetes or the kubernetes-SIG org](https://github.com/kubernetes/community/blob/master/community-membership.md) (see [notes](#cluster-api-release-team-vs-kuberneteskubernetes-sig-membership)).
 - Have some prior experience with contributing to CAPI releases (such having been in the release team for a prior release)
 - Have diverse company affiliations (i.e. not all from the same company)
+- Are members of the Kubernetes slack community (register if you are not!)
+- Are members of the Cluster Lifecycle SIG mailing list (subscribe if you are not!)
```

Comment:
suggestion: for complete newbie, this might not be super clear. Maybe we should link it to reference on how to do it the same [way](https://cluster-api.sigs.k8s.io/introduction.html#-community-discussion-contribution-and-support) we do in CAPI book